src/V3LinkInc.cpp

```diff
@@ -270,20 +288,23 @@ class LinkIncVisitor final : public VNVisitor {
             operp = new AstAdd{fl, readp->cloneTree(true), newconstp};
         }
 
+        AstAssign* assignp;
```

Comment:
Since not used outside the if, please declare it in both of the if/else branches.

Reply:
Done.